src/variation/hgvs_dup_del_mode.py

```diff
@@ -62,20 +81,20 @@ def default_mode(
 
     def copy_number_count_mode(
         self,
-        alt_type: AltType.DELETION
-        | AltType.DELETION_AMBIGUOUS
-        | AltType.DUPLICATION
-        | AltType.DUPLICATION_AMBIGUOUS,
+        alt_type: AltType,
         location: dict,
         baseline_copies: int,
     ) -> dict:
         """Return a VRS Copy Number Variation.
 
-        :param alt_type: The type of alteration
+        :param alt_type: The type of alteration. Must be one of ``DELS_DUPS``.
```

Comment:
can probably simplify 

```suggestion
        :param alt_type: The type of alteration. Must be ``DELS_DUPS``.
```